contracts/SpokePoolV3Periphery.sol

```diff
@@ -155,7 +230,7 @@ contract SpokePoolV3Periphery is Lockable, MultiCaller {
         uint32 fillDeadline,
         uint32 exclusivityParameter,
         bytes memory message
-    ) external payable nonReentrant {
```

Comment:
Not directly related to this specific diff, but is there a reason why you can not specify an explicit `outputToken` here?

Reply:
This is a copy/paste of the spoke pool verifier deposit function, so we just assume that this is only being used with the native token,